src/components/Cta.tsx

```diff
@@ -1,23 +1,22 @@
 import * as React from "react";
 import { twMerge } from "tailwind-merge";
 
-type Cta = {
+type Props = {
```

Comment:
Why'd you change the type? All our components should be consistent if their prop type names. Either all should be called Props (which I don't think you can do unless these are all private and not exported) or they should have the name of the component.

Reply:
Edit: I think they're private because I see you it's called Props in another component too. Should the breadcrumbs top level component be called Props then too?